netutils/webclient/webclient.c

```diff
@@ -724,6 +724,15 @@ int webclient_perform(FAR struct webclient_context *ctx)
         {
           char port_str[sizeof("65535")];
 
+#if defined(CONFIG_WEBCLIENT_NET_LOCAL)
+          if (ctx->unix_socket != NULL)
```

Comment:
If unix_socket doesn't implement TLS, why isn't conn.tls zero in the first place and then the whole block skip at line 723?

Reply:
to indicate the error to the user.